sources_nist_round3/kem/kyber/Reference_Implementation/crypto_kem/kyber512-90s/symmetric-shake.c

```diff
@@ -1,60 +0,0 @@
-#include <stddef.h>
```

Comment:
Similar to previous comment. Can we find a way to leave the sources 100% unaltered?